Framework/Algorithms/src/MuonGroupDetectors.cpp

```diff
@@ -98,17 +99,13 @@ void MuonGroupDetectors::exec() {
     outWS->getSpectrum(groupIndex).clearDetectorIDs();
 
     auto &Y = outWS->mutableY(groupIndex);
-    auto &E = outWS->mutableE(groupIndex);
 
     for (auto &wsIndex : wsIndices) {
-      for (size_t i = 0; i < inWS->blocksize(); ++i) {
-        // Sum the y values
-        Y[i] += inWS->y(wsIndex)[i];
-        // Sum the errors in quadrature
-        E[i] = sqrt(pow(E[i], 2) + pow(inWS->e(wsIndex)[i], 2));
-      }
-
-      // Detectors list of the group should contain all the detectors of it's
+	  outWS->mutableY(groupIndex) += inWS->y(wsIndex);
+	  inWS->mutableE(groupIndex) += inWS->e(wsIndex);
```

Comment:
`HistogramE` does not add the squares automatically! Add the histograms instead!